website/docs/Capabilities.md

```diff
@@ -240,49 +240,136 @@ When testing on Chrome, WebdriverIO will automatically download the desired brow
 ```ts
 {
     browserName: 'chrome',
-    browserVersion: '116' // or '116.0.5845.96', 'stable', 'dev', 'canary', 'beta'
+    browserVersion: '116.0.5845.96' // or 'stable', 'latest', 'dev', 'canary', 'beta'
```

Comment:
You can't use `116`. Ref:

https://github.com/webdriverio/webdriverio/blob/aa35c291fb496b80350766d674385ea7b4d1516a/packages/wdio-utils/src/driver/utils.ts#L138-L150

and running:

```js
await (async () => {
  const { canDownload, detectBrowserPlatform, resolveBuildId } = await import('@puppeteer/browsers')
  const browser = 'chrome'
  const platform = detectBrowserPlatform()

  const results = []
  for (const tag of ['116', '116.0.5845.96', 'stable', 'latest', 'dev', 'canary', 'beta']) {
    const buildId = await resolveBuildId(browser, platform, tag)
    const result = await canDownload({
      browser,
      platform,
      buildId,
    })

    results.push(`${tag}: ${result}`)
  }

  return results
})()
```

returns

```js
[
  '116: false',
  '116.0.5845.96: true',
  'stable: true',
  'latest: true',
  'dev: true',
  'canary: true',
  'beta: true'
]
```

Reply:
I ran the code using `@puppeteer/browsers` version `1.4.6` (used [here](https://github.com/webdriverio-community/node-geckodriver/blob/67926c471371ccdceb9a7abc16cf9ee8eab0aed8/package-lock.json#L1397)) that's why I got a different result from you.

Running it with `@puppeteer/browsers` version `1.7.0` (used [here](https://github.com/webdriverio/webdriverio/blob/aa35c291fb496b80350766d674385ea7b4d1516a/package-lock.json#L3588)) I get the same result as you. Will revert.